Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -1798,4 +1799,3 @@ bool NetworkStorageManager::shouldManageServiceWorkerRegistrationsByOrigin()
 #endif // ENABLE(SERVICE_WORKER)
 
 } // namespace WebKit
-
```

Comment:
It's an extra line. I think it's OK to remove it in this patch this time, though we usually try to avoid unrelated spacing changes in patches.

Reply:
Yeah, sorry, about that. I deleted my comment once I realized that there was in fact an extra line at the end.

Reply:
> though we usually try to avoid unrelated spacing changes in patches.

I'll keep in mind.